docs/network-flow-visibility.md

```diff
@@ -263,7 +263,9 @@ then please use the address:
 `<Ipfix-Collector Cluster IP>:<port>:<TCP|UDP>`
 * If you have deployed the [ELK
 flow collector](#deployment-steps-1), then please use the address:  
-`<Logstash Cluster IP>:4739:<TCP|UDP>`
+`<Logstash Cluster IP>:4739:<TCP|UDP>` for sending IPFIX messages, or `<Logstash Cluster IP>:4736:<TCP|UDP>`
+for sending JSON format records, with respect to record format specified in `recordFormat`
```

Comment:
for sending JSON format records. Record format is specified with `recordFormat` (defaults to IPFIX) and must match the format expected by the collector.